internal/controller/humiocluster_controller.go

```diff
@@ -70,6 +70,8 @@ const (
 	waitingOnPodsMessage = "waiting for pods to become ready"
 
 	humioVersionMinimumForReliableDownscaling = "1.173.0"
+
+	PDFRenderServiceImage = "humio/pdf-render-service:0.0.60--build-102--sha-c8eb95329236ba5fc65659b83af1d84b4703cb1e"
```

Comment:
Could to maybe elaborate on what this image is supposed to be? If the goal of this is to be the default image to use if the user didn't specify one, I suggest we put that into `internal/controller/versions/versions.go` where the other default versions are located.